README.md

```diff
@@ -12,13 +10,108 @@ Simply run:
 
 $ inspec plugin install train-habitat
 
+## Using the plugin as a Train Transport
+
+### Dual-mode transport
+
+Because habitat exposes some facts by its HTTP Gateway API, and some facts by its CLI tool `hab`, this Train Transport has three modes of operation:
+
+ * Using only the HTTP API (no ability to query packages, but rich ability to query rings)
+ * Using only the `hab` CLI command (limitations TBD)
+ * Using both (full capabilities)
+
+When creating a train-habitat Connection, there are thus two sets of options, prefixed with `api_` and `cli_` respectively. You must provide at least one set.
+
+### API-Mode options
+
```

Comment:
Is there supposed to be more context on how to configure the API-Mode options? I'm comparing this section with CLI Mode options.

Reply:
Shall I add examples?

Reply:
That would be wonderful! Another option is an overview explaining the API-Mode options under the API-Mode options header.